streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -298,7 +300,9 @@ private void prepareStreams() throws Exception {
         );
     }
 
-    private void prepareStreamThread(final StreamThread thread, final int threadId, final boolean terminable) throws Exception {
+    private void prepareStreamThread(final StreamThread thread,
```

Comment:
was this over 100?

Reply:
No, I think this was just left over from when I had added a new parameter which put it over 100, but I took it out when I found a better way to do the test.

Reply:
I have no problem leaving them on new lines, just didn't understand why when it didn't change the contents

Reply:
Yeah, sorry for the confusion